test/views/claimToAdjustPoa/CheckYourAnswersControllerViewSpec.scala

```diff
@@ -49,6 +49,7 @@ class CheckYourAnswersControllerViewSpec extends TestSupport {
       )
   }
 
+  //noinspection ScalaStyle
```

Comment:
Just wondering what was ScalaStyle error here?

Reply:
It was complaining that the method was too long